### DISCUSSION TITLE
TypeError: Cannot read properties of undefined (reading 'key')

### DISCUSSION THREAD

**Discussion by mariia-mitiureva** [inokawa/virtua#592] [Q&A] [answered]
After updating to 0.39.2 sometimes (quite rare) it crashes with the following error:

<img width="651" alt="image" src="https://github.com/user-attachments/assets/d4c5918c-cd17-429b-be36-71059cee23b1" />

I happens randomly, not sure what can be a trigger. Did it happen to someone else?

**Comment by inokawa** [comment]
Hi, are you using React 19?

**Reply by mariia-mitiureva** [reply]
No, 18.3.1
So far it happened only in dev mode

**Reply by mariia-mitiureva** [reply]
Note: I updated from `0.33.3`  to `0.39.2`

**Reply by inokawa** [reply]
Thanks.
Actually the same error happened in my prod with react@19 and virtua@0.39.0. Still not sure what's causing it...

**Comment by aeharding** [comment]
I think I found an end user report. I haven't been able to repro yet:

https://github.com/aeharding/voyager/issues/1799

In my bundle that is:

<img width="573" alt="image" src="https://github.com/user-attachments/assets/ec019626-4ff9-4303-b03c-f6e2d71ab91e" />

Seems to be https://github.com/inokawa/virtua/blob/7539c2a0aa279f4c2be6eb6c268344e22404df11/src/vue/utils.ts#L7

**Reply by aeharding** [reply]
@inokawa should this be converted to an issue?

**Reply by inokawa** [reply, marked as answer]
Thanks for the report. I created it #597
